fly/system/win/system_impl.cpp

```diff
@@ -64,12 +64,19 @@ int SystemImpl::GetErrorCode() noexcept
 //==============================================================================
 std::string SystemImpl::GetErrorString(int code) noexcept
 {
-    LPTSTR str = NULL;
+    LPTSTR str = nullptr;
     std::string ret;
 
-    ::FormatMessage(s_formatFlags, NULL, code, s_langId, (LPTSTR)&str, 0, NULL);
+    ::FormatMessage(
+        s_formatFlags,
+        nullptr,
+        code,
+        s_langId,
+        (LPTSTR)&str,
+        0,
+        nullptr);
 
-    if (str == NULL)
+    if (str == nullptr)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Condition 'str==nullptr' is always true](https://app.codacy.com/manual/trflynn89/libfly/pullRequest?prid=4779929)